recipes/ann/patches/cmake.patch

```diff
@@ -0,0 +1,629 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+new file mode 100644
+index 0000000..b11903d
+--- /dev/null
++++ b/CMakeLists.txt
+@@ -0,0 +1,20 @@
++cmake_minimum_required(VERSION 2.8)
++set(CMAKE_TRY_COMPILE_TARGET_TYPE "STATIC_LIBRARY")
++project(ann)
++IF(APPLE)
++    # Fix linking on 10.14+. See https://stackoverflow.com/questions/54068035
++    LINK_DIRECTORIES(/usr/local/lib)
```

Comment:
This looks like it's doing the wrong thing. Is it needed?

The build should use the 10.9 SDK for backwards compatibility which will be stored elsewhere (though normally the compiler wrappers handle it automatically).

Reply:
I had run into some linking errors when testing locally on Mojave, but it looks like that was fixed by one of the other command I added so I have removed this.